include/ibamr/KrylovLinearSolverStaggeredStokesSolverInterface.h

```diff
@@ -89,6 +89,12 @@ class KrylovLinearSolverStaggeredStokesSolverInterface : public StaggeredStokesS
      */
     void setVelocityPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& U_problem_coefs);
 
+	/*!
+	 * \brief Set if velocity and pressure have nullspace.
+	 */
+	void setComponentsHaveNullspace(const bool has_velocity_nullspace,
```

Comment:
> On Apr 24, 2015, at 12:33 AM, Amneet Bhalla <notifications@github.com> wrote:
> 
> In include/ibamr/KrylovLinearSolverStaggeredStokesSolverInterface.h <https://github.com/IBAMR/IBAMR/pull/6#discussion_r29023188>:
> 
> > @@ -89,6 +89,12 @@ class KrylovLinearSolverStaggeredStokesSolverInterface : public StaggeredStokesS
> >       */
> >      void setVelocityPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& U_problem_coefs);
> >  
> > +	/*!
> > +	 * \brief Set if velocity and pressure have nullspace.
> > +	 */
> > +	void setComponentsHaveNullspace(const bool has_velocity_nullspace,
> (1) I think so, because the implementations are capable of generating vectors themselves.
> (2) setComponentNullspaces() sounds like we are passing vectors, but I just to pass booleans..
> 
OK.
> —
> Reply to this email directly or view it on GitHub <https://github.com/IBAMR/IBAMR/pull/6/files#r29023188>.
>